packages/amplify_datastore/example/ios/unit_tests/DataStorePluginUnitTests.swift

```diff
@@ -37,11 +37,12 @@ var eventSentExp: XCTestExpectation?
 class DataStorePluginUnitTests: XCTestCase {
 
     var pluginUnderTest: SwiftAmplifyDataStorePlugin = SwiftAmplifyDataStorePlugin()
-    var flutterModelSchemaRegistration: FlutterModels = FlutterModels()
+    var modelSchemaRegistry = FlutterSchemaRegistry()
+    var customTypeSchemaRegistry = FlutterSchemaRegistry()
 
     override func setUpWithError() throws {
-        flutterModelSchemaRegistration.addModelSchema(modelName: "Post", modelSchema: testSchema)
-        flutterModelSchemaRegistration.registerModels(registry: ModelRegistry.self)
+        modelSchemaRegistry.addModelSchema(modelName: "Post", modelSchema: testSchema)
+        modelSchemaRegistry.registerModels(registry: ModelRegistry.self)
```

Comment:
Can we test custom types here?

Reply:
Similar to above explanation, this unit tests will invoke  `FlutterSerializedModel#toMap` which has been tested in its own test suite.